sci-mathematics/amath/amath-1.7.0.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="Simple command line calculator"
+DESCRIPTION="amath features a case sensitive command line interface, internal
+IEEE 754 calculations with 15 significant digits, calculations with real and
+complex numbers, variables and user defined functions, logarithmic and exponential \
+functions, trigonometric and hyperbolic function and selected mathematical \
+constants and rounding functions."
+HOMEPAGE="http://amath.innolan.net"
+LICENSE="BSD (2-clause)"
+COPYRIGHT="Carsten Sonne Larsen"
```

Comment:
@scottmc How to use the option of having it included in the tab and make the rest transparent ? I would like to try. Could you give a link or some other guideline ?